### PR TITLE
Add use-after-free advisory for `yottadb 1.1.0` and earlier

### PR DESCRIPTION
**Pull request by jyn514**
I've also yanked all affected versions.

### CODE REVIEW

**Comment by Shnatsel** [comment]
The advisory looks very detailed! Thanks for the report!

File: crates/yottadb/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "yottadb"
+date = "2021-02-09"
+url = "https://gitlab.com/YottaDB/Lang/YDBRust/-/issues/40"
+categories = ["memory-corruption"]
+keywords = ["use-after-free"]
+informational = "unsound"
```
Comment (Shnatsel) [inline crates/yottadb/RUSTSEC-0000-0000.md:9]: `informational = "unsound"` is used when the issue cannot occur in typical usage of the library. (We still need proper docs for it, which is #751). The provided code example looks fairly trivial to me, so if that causes memory corruption, I'd prefer to drop this field and file it as a regular advisory (i.e. vulnerability).

What do you think?

Comment (jyn514) [reply crates/yottadb/RUSTSEC-0000-0000.md:9]: Hmm, ok, seems reasonable. I agree this is pretty trivial, I'll drop the field.

**Comment by Shnatsel** [comment]
Thanks again!

**Pull request merged**